util/flags.py

```diff
@@ -29,6 +29,13 @@ def create_flags():
 
     f.DEFINE_float('augmentation_spec_dropout_keeprate', 1, 'keep rate of dropout augmentation on spectrogram (if 1, no dropout will be performed on spectrogram)')
 
+    f.DEFINE_boolean('augmentation_sparse_warp', False, 'whether to use spectrogram sparse warp')
```

Comment:
Given the performance characteristics of this augmentation, we can't currently use it in our models, so we won't be able to answer many questions about it. This is already the case with the rest of the augmentations, which is unfortunate. Are you willing to answer occasional issues about this code and review PRs made against it? If you're not, that's OK too, but then I'll ask you to add a note to the help string of this flag saying that it's unsupported, so people know that they're on their own with making it work.

Reply:
I will add note to say it's unsupported, because it's not really the same implementation like paper said, moreover, the paper also mentioned that time warping is not a major factor to improve test accuracy.